### PR TITLE
Real aggregate

### PR DESCRIPTION
**Pull request by dkaygithub**
completed aggregation UI

### CODE REVIEW

**Comment by ssaamm** [comment]
Squash these into 1 commit

File: js/chartjs/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Chart.js
```
Comment (ssaamm) [inline js/chartjs/README.md]: Do we need the Chart.js readme in our repo?

File: js/chartjs/docs/00-Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
```
Comment (ssaamm) [inline js/chartjs/docs/00-Getting-Started.md]: I don't think we need the entire `chartjs/docs` folder

File: js/chartjs/samples/doughnut.color.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<!doctype html>
```
Comment (ssaamm) [inline js/chartjs/samples/doughnut.color.html]: I don't think we need the entire `js/chartjs/samples` folder

File: js/chartjs/src/Chart.Bar.js [hunk 4]
```diff
@@ -0,0 +1,303 @@
+(function(){
```
Comment (ssaamm) [inline js/chartjs/src/Chart.Bar.js]: We probably don't need the source code...? Remove the `js/chartjs/src/` folder.

File: templates/userAggregate.html [hunk 5]
```diff
@@ -0,0 +1,230 @@
+{% extends "base.html" %}
+{% set active_page = "query" %}
+{% block title %}Reports{% endblock %}
+{% block head_extra %}
+        <script src="/buoy.git/js/chartjs/chart.min.js"></script>
```
Comment (ssaamm) [inline templates/userAggregate.html]: This should be `js/chartjs/chart.min.js`, no? Relative paths are important.

File: userAggregateSimple.php [hunk 6]
```diff
@@ -0,0 +1,116 @@
+<?php
```
Comment (ssaamm) [inline userAggregateSimple.php:1]: We don't need this file in here anymore, right? We have a (hopefully working) `userAggregate.php` now.

**Comment by dkaygithub** [comment]
i think it's ready

**Comment by ssaamm** [comment]
Squash the commits?

**Comment by dkaygithub** [comment]
how?
I know how to do it while i'm working on stuff, but after pushing i don't know how to do it in the ui

**Comment by ssaamm** [comment]
`git rebase -i HEAD~10`

**Comment by dkaygithub** [comment]
i will try, havent done command line in forever

**Pull request merged**